apimodels/test_results.go

```diff
@@ -27,29 +27,40 @@ type CedarTestResult struct {
 // GetCedarTestResultsOptions represents the arguments passed into the
 // GetCedarTestResults function.
 type GetCedarTestResultsOptions struct {
-	BaseURL   string
-	TaskID    string
-	TestName  string
-	Execution int
+	BaseURL       string
+	TaskID        string
+	DisplayTaskID string
+	TestName      string
+	Execution     int
 }
 
 // GetCedarTestResults makes request to cedar for a task's test results.
 func GetCedarTestResults(ctx context.Context, opts GetCedarTestResultsOptions) ([]CedarTestResult, error) {
-	getOpts := timber.GetOptions{
-		BaseURL:   fmt.Sprintf("https://%s", opts.BaseURL),
-		TaskID:    opts.TaskID,
-		TestName:  opts.TestName,
-		Execution: opts.Execution,
+	getOpts := testresults.TestResultsGetOptions{
+		CedarOpts: timber.GetOptions{
+			BaseURL: fmt.Sprintf("https://%s", opts.BaseURL),
```

Comment:
Is there a reason we don't require auth for fetching test results (and other routes like historical test data/system metrics)?

Reply:
The only thing cedar requires auth for are logs, nothing is else is protected.

Reply:
If you are asking why cedar only requires auth for logs, then it is because we don't actually have authentication in cedar and use evergreen as a proxy. So for things we don't care to protect, it isn't worth it.

Reply:
We resolved this offline - it's fine to send the test results since it's mostly metadata (the test logs require auth).